### PR TITLE
docs: Update arm-performance-google-aws-azure-with-apisix.md

### PR DESCRIPTION
**Pull request by Yilialinn**
Fixes: #[Add issue number here]

Changes:

The comparison with Oracle is increased based on the former passage. 

Rendered Version: https://github.com/Yilialinn/apisix-website/blob/master/blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md

### CODE REVIEW

**Comment by Yilialinn** [comment]
@juzhiyuan 请确认

File: blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md [hunk 1]
```diff
@@ -1,129 +1,132 @@
...
+[![Apache APISIX's Architecture Diagram](https://static.apiseven.com/2022/10/21/635241c9d2c35.jpeg)](https://static.apiseven.com/2022/10/21/635241c9d2c35.jpeg)
 
-### Test Results
+We use Apache APISIX to bind a single CPU on AWS c7g.large, GCP t2a-standard-2, Azure D2ps v5 (although the name includes D2ps, but it is a dual-core CPU belonging to the Dpsv5 series), and OCI A1 to conduct stress testing and analyze the performance of the server through QPS and response latency.
 
-The figure below is the QPS (queries per second) test result and the higher the number, the better the performance.
+We use  [Apache APISIX’s official open-source performance benchmark](https://github.com/apache/apisix/blob/master/benchmark/run.sh) for testing.
 
-![QPS result](https://static.apiseven.com/2022/blog/0812/5.png)
+### Test Cases
```
Comment (guoqqqi) [inline blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md:89]:
```suggestion
### Test Cases

```

File: blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md [hunk 2]
```diff
@@ -1,129 +1,133 @@
...
 
-The latest AWS Graviton3 processors to be released at the end of May 2022 are based on the more advanced [Neoverse V1](https://www.arm.com/zh-TW/products/silicon-ip-cpu/neoverse/neoverse-v1) design, they offer up to twice the floating point performance, twice the cryptographic performance, and three times the ML compared to the AWS Graviton2 processors performance, including support for bfloat16. The following figure shows the main models equipped with AWS Graviton3 processors:
+The following figure shows the main models equipped with AWS Graviton3 processors:
 
-![AWS Graviton3 processors](https://static.apiseven.com/2022/blog/0812/1.png)
+[![AWS Graviton3 processors](https://static.apiseven.com/2022/10/21/6352412740665.webp)](https://static.apiseven.com/2022/10/21/6352412740665.webp)
```
Comment (guitu168) [inline blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md]:
```suggestion
![AWS Graviton3 processors](https://static.apiseven.com/2022/10/21/6352412740665.webp)
```

File: blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md [hunk 3]
```diff
@@ -1,129 +1,133 @@
...
+- **Integration with Google Cloud services**: T2A VMs support the most popular Linux operating systems such as RHEL, Suse Linux Enterprise Server, CentOS, Ubuntu, and Rocky Linux. In addition, T2A VMs also support Container-optimized OS to bring up Docker containers quickly, efficiently, and securely. Further, developers building applications on Google Cloud can already use several Google Cloud services with T2A VMs.
+- **Extensive ISV partner ecosystem**: Ampere lists more than 100 applications, databases, cloud-native software, and programming languages that are already running on Ampere-based T2A VMs, with more being added all the time.
 
-![Tau T2A VM](https://static.apiseven.com/2022/blog/0812/2.png)
+The main models are as follows:
 
-### Azure Arm-based Virtual Machines
+[![Google Cloud Platform T2A Models](https://static.apiseven.com/2022/10/21/6352412815275.webp)](https://static.apiseven.com/2022/10/21/6352412815275.webp)
```
Comment (guitu168) [inline blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md]:
```suggestion
![Google Cloud Platform T2A Models](https://static.apiseven.com/2022/10/21/6352412815275.webp)
```

File: blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md [hunk 4]
```diff
@@ -1,129 +1,133 @@
...
+### Azure ARM-based Virtual Machines
 
-![Dpsv5 and Epsv5 VMs](https://static.apiseven.com/2022/blog/0812/3.png)
+In April 2022, Microsoft announced a preview of its family of Azure virtual machines based on Ampere® Altra® Arm processors. The new VMs are designed to efficiently run scale-out workloads, web servers, application servers, open-source databases, cloud-native and rich .NET applications, Java applications, game servers, media servers, and more. The new VM series includes general-purpose Dpsv5 and memory-optimized Epsv5 VMs. The main models are as follows:
 
-## Three cloud vendors ARM server performance test
+[![Azure ARM-based Virtual Machines](https://static.apiseven.com/2022/10/21/635241c219ef7.jpeg)](https://static.apiseven.com/2022/10/21/635241c219ef7.jpeg)
```
Comment (guitu168) [inline blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md]:
```suggestion
![Azure ARM-based Virtual Machines](https://static.apiseven.com/2022/10/21/635241c219ef7.jpeg)
```

File: blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md [hunk 5]
```diff
@@ -1,129 +1,133 @@
...
 
-We will test the performance of Apache APISIX under two typical scenarios in order to obtain more realistic and rich test data:
+Apache APISIX is a cloud-native, high-performance, scalable, open-source API gateway. Compared with traditional API gateways, Apache APISIX is developed based on NGINX and LuaJIT, with features such as dynamic routing and plugin hot reloading, which is very suitable for API management under cloud-native architecture. The architecture diagram is shown below:
 
-* **Scenario 1: Single upstream.** In this scenario, a single upstream (without any plugins) is used to test the performance of APISIX in pure proxy back-to-origin mode.
+[![Apache APISIX's Architecture Diagram](https://static.apiseven.com/2022/10/21/635241c9d2c35.jpeg)](https://static.apiseven.com/2022/10/21/635241c9d2c35.jpeg)
```
Comment (guitu168) [inline blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md]:
```suggestion
![Apache APISIX's Architecture Diagram](https://static.apiseven.com/2022/10/21/635241c9d2c35.jpeg)
```

File: blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md [hunk 6]
```diff
@@ -1,129 +1,133 @@
...
 
-| VM series / vCPU | 1       | 2       | 4       | 8      | 16      | 32      | 64      |
-|------------------|---------|---------|:--------|:-------|:--------|:--------|:--------|
-| AWS C7g          | $0.0361 | $0.0723 | $0.1445 | $0.289 | $0.5781 | $1.1562 | $1.7342 |
-| GCP T2A          | $0.0385 | $0.077  | $0.154  | $0.308 | $0.616  | $1.232  | $1.848  |
-| Azure Dpsv5      | *       | $0.077  | $0.154  | $0.308 | $0.616  | $1.232  | $1.848  |
+The figure below is the response latency test results in milliseconds. The smaller the value, the better the performance.
 
-The following table summarizes the cost and cost–performance ratio of AWS c7g.large and GCP t2a-standard-2 running for one year, referring to the QPS data for a single upstream in the Apache APISIX performance test. The larger the number, the higher the QPS can be obtained at the unit price.
+[![Response Latency of AWS c7g, GCP, Azure and OCI A1](https://static.apiseven.com/2022/10/21/635241298c145.webp)](https://static.apiseven.com/2022/10/21/635241298c145.webp)
```
Comment (guitu168) [inline blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md]:
```suggestion
![Response Latency of AWS c7g, GCP, Azure and OCI A1](https://static.apiseven.com/2022/10/21/635241298c145.webp)
```

File: blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md [hunk 7]
```diff
@@ -1,129 +1,133 @@
...
 
-* **Scenario 1: Single upstream.** In this scenario, a single upstream (without any plugins) is used to test the performance of APISIX in pure proxy back-to-origin mode.
+[![Apache APISIX's Architecture Diagram](https://static.apiseven.com/2022/10/21/635241c9d2c35.jpeg)](https://static.apiseven.com/2022/10/21/635241c9d2c35.jpeg)
 
-* **Scenario 2: Single upstream + multiple plugins.** This scenario uses a single upstream with multiple plugins and two plugins are used here. It mainly tests the performance of APISIX when the two core consumption performance plugins, `limit-count` and `prometheus`, are enabled.
+We use Apache APISIX to bind a single CPU on AWS c7g.large, GCP t2a-standard-2, Azure D2ps v5 (although the name includes D2ps, but it is a dual-core CPU belonging to the Dpsv5 series), and OCI A1 to conduct stress testing and analyze the performance of the server through QPS and response latency.
 
-### Test Results
+We use  [Apache APISIX’s official open-source performance benchmark](https://github.com/apache/apisix/blob/master/benchmark/run.sh) for testing.
```
Comment (yzeng25) [inline blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md]:
```suggestion
We use [Apache APISIX’s official open-source performance benchmark](https://github.com/apache/apisix/blob/master/benchmark/run.sh) for testing.
```

File: blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md [hunk 8]
```diff
@@ -1,129 +1,133 @@
...
+This scenario uses a single upstream with two plugins. It mainly tests the performance of APISIX when the two core consumption performance plugins, `limit-count` and `prometheus`, are operating.
 
-From the perspective of QPS and response delay, under network IO-intensive API gateways like Apache APISIX, AWS C7g has a 100% performance improvement compared to GCP T2A, and Azure Dpsv5 has a performance lead of about 15% compared to GCP T2A.
+### Test Results
 
-## Cost-performance comparison
+The figure below is the QPS (queries per second) test result of AWS c7g.large, GCP t2a-standard-2, Azure D2ps v5, and OCI A1. The higher the QPS value, the better the performance of the server.
 
-Since this article only focuses on testing the performance of ARM machines from different cloud vendors, so we will ignore the change of "the same number of CPU cores with different memory", and analyze the cost-performance ratio of AWS Graviton3 and GCP T2A only from the perspective of the number of CPU cores.
+[![QPS Value Comparison of AWS c7g, GCP, Azure and OCI A1](https://static.apiseven.com/2022/10/21/635241290d787.webp)](https://static.apiseven.com/2022/10/21/635241290d787.webp)
```
Comment (yzeng25) [inline blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md]: Missed one cc @hf400159 
```suggestion
![QPS Value Comparison of AWS c7g, GCP, Azure and OCI A1](https://static.apiseven.com/2022/10/21/635241290d787.webp)
```

File: blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md (hunk 6 above)
Comment (yzeng25) [inline blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md]: Ditto. When inserting an image/picture, the correct form should be 

`image/picture name in text`

```suggestion
![Response Latency of AWS c7g, GCP, Azure and OCI A1](https://static.apiseven.com/2022/10/21/635241298c145.webp)
```

File: blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md [hunk 9]
```diff
@@ -1,129 +1,133 @@
...
+The ARM architecture is a member of the [RISC (Reduced instruction set computer)](https://en.wikipedia.org/wiki/Reduced_instruction_set_computer) design family. The RISC microprocessor architecture design enables small processors to efficiently handle complex tasks by using a set of highly optimized instructions. Being widely used in many embedded system designs, the ARM architecture has become the cornerstone of the world’s largest computing ecosystem and mobile devices. Many experts regard it as the future of cloud computing due to its advantages of low power consumption, low cost, high performance, and flexible licensing. Therefore, mainstream cloud vendors led by AWS (Amazon Web Services), GCP (Google Cloud Platform), Azure (Microsoft Azure), and Oracle have successively launched ARM-based servers. This article selects servers from these vendors to conduct performance testing. Let’s first examine the four major manufacturers and their products.
```
Comment (yzeng25) [inline blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md]:
```suggestion
The ARM architecture is a member of the [RISC (Reduced instruction set computer)](https://en.wikipedia.org/wiki/Reduced_instruction_set_computer) design family. The RISC microprocessor architecture design enables small processors to efficiently handle complex tasks by using a set of highly optimized instructions. Being widely used in many embedded system designs, the ARM architecture has become the cornerstone of the world’s largest computing ecosystem and mobile devices. Many experts regard it as the future of cloud computing due to its advantages of low power consumption, low cost, high performance, and flexible licensing. Therefore, mainstream cloud vendors led by AWS (Amazon Web Services), GCP (Google Cloud Platform), Azure (Microsoft Azure), and OCI (Oracle Cloud Infrastructure) have successively launched ARM-based servers. This article selects servers from these vendors to conduct performance testing. Let’s first examine the four major manufacturers and their products.
```

File: blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md [hunk 10]
```diff
@@ -1,129 +1,133 @@
...
 
-They offer up to 32 Gbps of network bandwidth and a wide range of network-attached storage options, making the Tau T2A VM suitable for scale-out workloads including web servers, containerized microservices, data record processing, media transcoding, and Java applications. The main models are as follows:
+- **Integration with Google Cloud services**: T2A VMs support the most popular Linux operating systems such as RHEL, Suse Linux Enterprise Server, CentOS, Ubuntu, and Rocky Linux. In addition, T2A VMs also support Container-optimized OS to bring up Docker containers quickly, efficiently, and securely. Further, developers building applications on Google Cloud can already use several Google Cloud services with T2A VMs.
```
Comment (yzeng25) [inline blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md]:
```suggestion
- **Integration with Google Cloud services**: T2A VMs support the most popular Linux operating systems such as RHEL, SUSE Linux Enterprise Server, CentOS, Ubuntu, and Rocky Linux. In addition, T2A VMs also support Container-optimized OS to bring up Docker containers quickly, efficiently, and securely. Further, developers building applications on Google Cloud can already use several Google Cloud services with T2A VMs.
```

File: blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md [hunk 11]
```diff
@@ -1,129 +1,133 @@
...
 
-AWS launched the first ARM-based processor, AWS Graviton, in 2018. It was about 4 years ahead of GCP for the deployment of the ARM-based server field. Now the AWS Graviton processor has developed to the third generation.
+## Conclusion
 
-Through the performance test results and price–performance ratio analysis of Apache APISIX, we can see that AWS Graviton3 has higher performance and cost-effective than GCP T2A and Azure Dpsv5. This is inseparable from the fact that AWS has been deeply involved in the field of ARM-based servers for many years.
+Through the analysis of the performance test results of Apache APISIX, we can see that AWS Graviton3 has higher performance than GCP T2A, Azure Dpsv5, and OCI A1. However, we only used the Apache APISIX binding single-core test during the test. The performance presented by the four may be different if multi-core is used.
 
-In addition, we used Apache APISIX bound single-core tests during our testing, and the price-performance ratio of AWS Graviton 3 may be further improved if multiple cores are used.
+There will have more tests using multi-core in the future, so stay tuned!
```
Comment (yzeng25) [inline blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md]:
```suggestion
We will reveal multi-core test results in the future, please stay tuned!
```

File: blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md [hunk 12]
```diff
@@ -1,129 +1,133 @@
...
+- **AWS Graviton1** processors feature custom silicon and 64-bit Neoverse cores.
+- **AWS Graviton2**-based instances support a wide range of general purpose, burstable, compute-optimized, memory-optimized, storage-optimized, and accelerated computing workloads, including application servers, microservices, high-performance computing (HPC), CPU-based machine learning (ML) inference, video encoding, electronic design automation, gaming, open-source databases, and in-memory caches. In order to provide a one-stop service experience, many AWS services also support Graviton2-based instances.
+- **AWS Graviton3** processors are the latest in the AWS Graviton processor family. They provide up to 25% better compute performance, 2x higher floating-point performance, and up to 2x faster cryptographic workload performance compared to AWS Graviton2 processors. AWS Graviton3 processors deliver 3x better performance compared to AWS Graviton2 processors for ML workloads, including support for bfloat16. They also support DDR5 memory, which provides 50% more memory bandwidth compared to DDR4.
```
Comment (yzeng25) [inline blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md]:
```suggestion
- **AWS Graviton3** processors are the latest in the AWS Graviton processor family. They provide up to 25% better compute performance, **2 times** higher floating-point performance, and up to **2 times** faster cryptographic workload performance compared to AWS Graviton2 processors. AWS Graviton3 processors deliver **3 times** better performance compared to AWS Graviton2 processors for ML workloads, including support for bfloat16. They also support DDR5 memory, which provides **50%** more memory bandwidth compared to DDR4.
```

Comment (yzeng25) [reply blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md]: I think using bold letters can better emphasis the sentence.

**Comment by Yilialinn** [comment]
@juzhiyuan @guoqqqi Problems are all solved. Please check the final version.

**Review by juzhiyuan — APPROVED** [review]
Just checked previous comments.

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**